py-utils/src/utils/msg_bus/message_bus/adaptee/adaptee.py

```diff
@@ -0,0 +1,22 @@
+
+
+class Adaptee(object):
+    def __init__(self):
+        pass
+
+    def send(self):
+        pass
+
+    def receive(self):
```

Comment:
Please add parameters to the interfaces, even though its base abstract class

Reply:
Params added